get-started/ch1.md

```diff
@@ -1,143 +1,266 @@
+<!-- Even if you've already written a lot of JS before, this book should not be skimmed over or skipped; take your time to fully process the material here. **A good start always depends on a solid first step.** -->
 
-For example, we use the `{` to begin a block of code and the `}` to end that block of code, just like C/C++ and Java. We also use the `;` to punctuate the end of a statement.
+## Qu'y a-t-il avec ce nom ?
 
-In some ways, legal relationships run even deeper than the syntax. Oracle (via Sun), the company that still owns and runs Java, also owns the official trademark for the name "JavaScript" (via Netscape). This trademark is almost never enforced, and likely couldn't be at this point.
+Le nom JavaScript est probablement le plus mal compris des noms de langages de programmation.
 
-For these reasons, some have suggested we use JS instead of JavaScript. That is a very common shorthand, if not a good candidate for an official language branding itself. Indeed, these books use JS almost exclusively to refer to the language.
+<!-- The name JavaScript is probably the most mistaken and misunderstood programming language name. -->
 
-Further distancing the language from the Oracle-owned trademark, the official name of the language specified by TC39 and formalized by the ECMA standards body is **ECMAScript**. And indeed, since 2016, the official language name has also been suffixed by the revision year; as of this writing, that's ECMAScript 2019, or otherwise abbreviated ES2019.
+Est-il lié à Java ? Est-ce uniquement la version scriptée de Java ? Est-ce uniquement pour écrire des scripts et non de réels programmes ?
 
-In other words, the JavaScript/JS that runs in your browser or in Node.js, is *an* implementation of the ES2019 standard.
+<!-- Is this language related to Java? Is it only the script form for Java? Is it only for writing scripts and not real programs? -->
+
+La vérité est que le nom JavaScript est un artefact une combine marketing. Quand Brendan Eich a d'abord conçu le langage, il l'a appelé Mocha. En interne à Netscape, la marque LiveScript était utilisée. Mais quand il a fallu le nommé publiquement, "JavaScript" remporta les suffrages.
+
+<!-- The truth is, the name JavaScript is an artifact of marketing shenanigans. When Brendan Eich first conceived of the language, he code-named it Mocha. Internally at Netscape, the brand LiveScript was used. But when it came time to publicly name the language, "JavaScript" won the vote. -->
+
+Pourquoi ? Car le langage est originellement designé pour attirer une audience principalement composée de développeurs Java et parce que le mot "script" était populaire à ce moment, faisant référent à des programmes très légers. Ces programmes ultra légers seraient les premier à s'insérer dans les pages de cette nouvelle chose appelée web !
```

Comment:
```suggestion
Pourquoi ? Car le langage est originellement désigné pour attirer une audience principalement composée de développeurs Java et parce que le mot "script" était à ce moment là populaire, faisant référence à des programmes très légers. Ces programmes ultra légers seraient les premier à s'insérer dans les pages de cette nouvelle chose appelée web !
```

Reply:
Here I changed a bit more but thanks for suggestions !